tsconfig.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": "1.5.0-beta",
+    "compilerOptions": {
+        "declaration": true,
+        "module": "commonjs",
+        "noImplicitAny": true,
+        "outDir": "_build",
+        "target": "es5"
+    },
+    "compileOnSave": false,
+    "filesGlob": [
+        "./**/*.ts",
+        "!./node_modules/**/*.ts",
+        "!./_build/**/*.ts",
+        "!./*.ts"
+    ],
+    "files": [
+        "./src/arrayFind.ts",
```

Comment:
`filesGlob` is for `atom-typescript`. My other goal is to write a tool which will look for that key and then regenerate the `files` array. It's extremely annoying to maintain the files array by hand. TypeScript supports not using it but lots of tools fill it in automatically.